ai_ta_backend/rabbitmq/ingest.py

```diff
@@ -63,10 +63,11 @@ class Ingest:
     """
 
     def __init__(self):
-        self.openai_api_key = os.getenv('OPENAI_API_KEY')
-        self.openai_api_base = os.environ['EMBEDDING_API_BASE'] + "/embeddings" if os.environ['EMBEDDING_API_BASE'] else None
-        self.ncsa_hosted_api_key = os.getenv('NCSA_HOSTED_API_KEY')
-        self.embedding_model = os.environ['EMBEDDING_MODEL']
+        self.illinois_chat_config = os.getenv('ILLINOIS_CHAT_CONFIG', 'false')
+        self.openai_api_key = self.illinois_chat_config == 'false' and os.getenv('OPENAI_API_KEY')
+        self.openai_api_base = os.environ['EMBEDDING_API_BASE'] + "/embeddings" if os.environ['EMBEDDING_API_BASE'] else 'https://api.openai.com/v1/embeddings'
+        self.ncsa_hosted_api_key = self.illinois_chat_config == 'false' and os.getenv('NCSA_HOSTED_API_KEY') or self.openai_api_key
```

Comment:
This inline logic and and or logic is a bit hard to follow, is there possible way to rewrite this to be more readable?